topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -1129,37 +1335,79 @@ Here, treatment is the primary factor that we are interested in. The sequencing
->        - In *"1: Factor level"*:
->          - *"Specify a factor level"*: `PE`
->          - {% icon param-files %} *"Counts file(s)"*: the 4 gene count files with `paired` in their name
->        - In *"2: Factor level"*:
->          - *"Specify a factor level"*: `SE`
->          - {% icon param-files %} *"Counts file(s)"*: the 3 gene count files with `single` in their name
->    - *"Files have header?"*: `No`
->    - *"Visualising the analysis results"*: `Yes`
->    - *"Output normalized counts table"*: `Yes`
->
->    {% snippet faqs/galaxy/tools_select_multiple_datasets.md %}
+>        - In *"Factor"*:
+>            - {% icon param-repeat %} *"Insert Factor"*
+>                - *"Specify a factor name, e.g. effects_drug_x or cancer_markers"*: `Treatment`
+>                - In *"Factor level"*:
+>                    - {% icon param-repeat %} *"Insert Factor level"*
+>                        - *"Specify a factor level, typical values could be 'tumor', 'normal', 'treated' or 'control'"*: `treated`
+>                        - {% icon param-collection %} *"Counts file(s)"*: the collection `treat`
+>                    - {% icon param-repeat %} *"Insert Factor level"*
+>                        - *"Specify a factor level, typical values could be 'tumor', 'normal', 'treated' or 'control'"*: `untreated`
+>                        - {% icon param-collection %} *"Counts file(s)"*: the collection `untreat`
+>            - {% icon param-repeat %} *"Insert Factor"*
+>                - *"Specify a factor name, e.g. effects_drug_x or cancer_markers"*: `Sequencing`
+>                - In *"Factor level"*:
+>                    - {% icon param-repeat %} *"Insert Factor level"*
+>                        - *"Specify a factor level, typical values could be 'tumor', 'normal', 'treated' or 'control'"*: `PE`
+>                        - {% icon param-collection %} *"Counts file(s)"*: the collection `paired`
+>                    - {% icon param-repeat %} *"Insert Factor level"*
+>                        - *"Specify a factor level, typical values could be 'tumor', 'normal', 'treated' or 'control'"*: `SE`
+>                        - {% icon param-collection %} *"Counts file(s)"*: the collection `single`
+>    - *"Files have header?"*: `Yes`
+>    - *"Choice of Input data"*: `Count data (e.g. from HTSeq-count, featureCounts or StringTie)`
+>    - In *"Output options"*:
+>        - *"Output selector"*: `Generate plots for visualizing the analysis results`, `Output normalised counts`
 >
 >    > ### {% icon comment %} Comment: Using group tags for large sample sets
 >    >
->    > If you have a large number of samples, or a complex experimental design, manually selecting files for each factor level
->    > may be a lot of work. In these situations, **group tags** can make your life a lot easier.
+>    > If you have a large number of samples, or a complex experimental design, an alternative to collections is **group tags**.
 >    >
```

Comment:
The new image seems to be too small; I think that it would be nice to increase the resolution.

![Screenshot from 2022-04-14 12-41-08](https://user-images.githubusercontent.com/26678117/163372587-a18f6e3b-0b69-4c2a-93b5-1da22d1fc665.png)

Reply:
Same for this one:

![Screenshot from 2022-04-14 12-42-12](https://user-images.githubusercontent.com/26678117/163373266-4a39aa4c-fca9-4295-b418-ef4e98913600.png)

Reply:
This happened when I converted pdf to png. I will redo the conversion.

Reply:
Thanks!